cgmanifest.json

```diff
@@ -8051,7 +8051,7 @@
         "type": "other",
         "other": {
           "name": "kata-containers",
-          "version": "3.2.0.azl0",
+          "version": "3.2.0.azl1",
           "downloadUrl": "https://github.com/microsoft/kata-containers/archive/refs/tags/3.2.0.azl0.tar.gz"
```

Comment:
```suggestion
          "downloadUrl": "https://github.com/microsoft/kata-containers/archive/refs/tags/3.2.0.azl1.tar.gz"
```

Reply:
Thanks! Unfortunately automation nulls the downloadUrl https://github.com/microsoft/azurelinux/pull/9005/commits/a1c18126e7dcc2c5f03ca82897fe26e9a4549a73#diff-45a0a38d395bf4427d0291d9c72bd99b57fa43e14b00e059ae170d657949f1f2R8065